### PR TITLE
added `@Bulkhead` and `@Timeout` non-negative diagnostic support

### PR DESCRIPTION
**Pull request by AlexXuChen**
Added mp-fault-tolerance `@Bulkhead` and `@Timeout` non-negative value diagnostic support.

References #77 

Signed-off-by: Alexander Chen <alchen@redhat.com>

### CODE REVIEW

**Comment by AlexXuChen** [comment]
Blocked by #188

File: microprofile.jdt/org.eclipse.lsp4mp.jdt.core/plugin.xml
Comment (rgrunber) [inline microprofile.jdt/org.eclipse.lsp4mp.jdt.core/plugin.xml:140]: I would still remove this attribute as it causes the entire validation to fail for Bulkhead.

File: microprofile.jdt/org.eclipse.lsp4mp.jdt.core/plugin.xml
Comment (rgrunber) [inline microprofile.jdt/org.eclipse.lsp4mp.jdt.core/plugin.xml:147]: Just need to adjust the indent here so it matches the rest of the file. After that, we can merge.

**Review by rgrunber — APPROVED** [review]
Looks good, just minor adjustment in the plugin.xml

**Pull request merged**
